test/e2e/network/conntrack.go

```diff
@@ -103,16 +104,25 @@ var _ = SIGDescribe("Conntrack", func() {
 				len(nodes.Items))
 		}
 
-		ips := e2enode.CollectAddresses(nodes, v1.NodeInternalIP)
+		ips := e2enode.GetAddresses(&nodes.Items[0], v1.NodeInternalIP)
 
 		clientNodeInfo = nodeInfo{
 			name:   nodes.Items[0].Name,
 			nodeIP: ips[0],
 		}
 
+		ips = e2enode.GetAddresses(&nodes.Items[1], v1.NodeInternalIP)
+
 		serverNodeInfo = nodeInfo{
 			name:   nodes.Items[1].Name,
-			nodeIP: ips[1],
+			nodeIP: ips[0],
+		}
+
+		for _, ip := range e2enode.GetAddresses(&nodes.Items[1], v1.NodeInternalIP) {
```

Comment:
I think that we are almost there, but I feel that is better that we use the right ip for both clientNodeInfo and serverNodeInfo, maybe just do the assignment directly instead of modifying it later ?